PAYMENTS.md

```diff
@@ -0,0 +1,28 @@
+# Integrate your Stripe account to accept your user's payments
+All the information that you need to setup your Stripe account into OSEM.
+If you have any problems with installing don't hesitate to [contact us](https://github.com/openSUSE/osem#contact)
+
+## Configure Stripe into the application
+To configure Stripe into your application all you need to do is add the private and publishable keys into the Rails environment.
+
+Add your Stripe API keys in `config/secrets.yml` into these variables:
+
+If you are using the application in development mode your config should look like this:  
+  `STRIPE_PUBLISHABLE_KEY = 'pk_**test**_random123example456'`  
+  `STRIPE_SECRET_KEY = 'sk_**test**_random123example456'`
+
+Otherwise, while in production mode, it should look like:  
+  `STRIPE_PUBLISHABLE_KEY = 'pk_**live**_random123example456'`  
+  `STRIPE_SECRET_KEY = 'sk_**live**_random123example456'`
```

Comment:
these instructions are a bit confusing because you refer to `secrets.yml`, but that file uses all lower-case letters:

![screenshot from 2016-08-04 16-15-09](https://cloud.githubusercontent.com/assets/2944/17405193/d760be18-5a5e-11e6-9a10-3a072efc2ca3.png)